tests/unit/Storefront/Theme/ThemeScriptsTest.php

```diff
@@ -81,6 +87,11 @@ public function testLoadPaths(): void
         $request->attributes->set(PlatformRequest::ATTRIBUTE_SALES_CHANNEL_ID, 'Storefront');
         $request->attributes->set(SalesChannelRequest::ATTRIBUTE_THEME_ID, 'Storefront');
         $request->attributes->set(SalesChannelRequest::ATTRIBUTE_THEME_NAME, 'Storefront');
+        $request->attributes->set(PlatformRequest::ATTRIBUTE_CONTEXT_OBJECT, Context::createCLIContext());
+        $salesChannelContext = $this->createMock(SalesChannelContext::class);
```

Comment:
instead of creating a mock, this helper could also be used: `\Shopware\Core\Test\Generator::createSalesChannelContext`